internal/page/certificateprovider/what_is_your_home_address.go

```diff
@@ -0,0 +1,108 @@
+package certificateprovider
+
+import (
+	"context"
+	"errors"
+	"net/http"
+
+	"github.com/ministryofjustice/opg-go-common/template"
+	"github.com/ministryofjustice/opg-modernising-lpa/internal/form"
+	"github.com/ministryofjustice/opg-modernising-lpa/internal/page"
+	"github.com/ministryofjustice/opg-modernising-lpa/internal/place"
+	"github.com/ministryofjustice/opg-modernising-lpa/internal/validation"
+)
+
+type whatIsYourHomeAddressData struct {
+	App       page.AppData
+	Addresses []place.Address
+	Form      *form.AddressForm
+	Errors    validation.List
+}
+
+func WhatIsYourHomeAddress(logger Logger, tmpl template.Template, addressClient AddressClient, donorStore DonorStore, certificateProviderStore CertificateProviderStore) page.Handler {
+	return func(appData page.AppData, w http.ResponseWriter, r *http.Request) error {
+		lpa, err := donorStore.GetAny(r.Context())
```

Comment:
this isn't used for anything (the `Redirect` method needs refactoring at some point, taking the Lpa has been confusing since we added the other actors, but it only does anything for donor routes)